.github/workflows/build.yml

```
@@ -18,25 +18,27 @@ jobs:
         # with:
         # submodules: "recursive" # Ensures submodules are checked out
 
-      - name: Set up Node.js
-        uses: actions/setup-node@v4
+      - name: Set up Deno
+        uses: denoland/setup-deno@v2
         with:
-          node-version: 20.10.0
+          deno-version: v2.x
 
       - name: Build
         run: |
-          yarn
-          yarn build
-        env: # Set environment variables for the build
-          SUPABASE_URL: "https://wfzpewmlyiozupulbuur.supabase.co"
-          SUPABASE_ANON_KEY: "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJzdXBhYmFzZSIsInJlZiI6IndmenBld21seWlvenVwdWxidXVyIiwicm9sZSI6ImFub24iLCJpYXQiOjE2OTU2NzQzMzksImV4cCI6MjAxMTI1MDMzOX0.SKIL3Q0NOBaMehH0ekFspwgcu3afp3Dl9EDzPqs1nKs"
+          export NODE_ENV="${NODE_ENV:-production}"
+          export SUPABASE_URL="${SUPABASE_URL:-https://placeholder.supabase.co}"
```

Shouldn't the build fail instead of having defaults here?